tg_bot/modules/locks.py

```diff
@@ -80,6 +83,59 @@ def locktypes(bot: Bot, update: Update):
     update.effective_message.reply_text("\n - ".join(["Locks: "] + list(LOCK_TYPES) + list(RESTRICTION_TYPES)))
 
 
+@user_admin
+def add_whitelist(bot: Bot, update: Update):
```

Comment:
This should be `@run_async` to ensure as few blocks as possible
Also needs to be `@loggable` to ensure log channels get info -> which means returning a log at the end as well